### PR TITLE
IP 17

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: unabridged-works-volume-4/original/17-letters.adoc [hunk 1]
```diff
@@ -392,11 +420,15 @@ but mayst meet with the right key, and the right understanding,
 wherein thy soul will be safe and happy; which is my heart`'s desire for thee,
 who am a real friend and well-wisher to thy soul in the Lord.
 
+[.signed-section-signature]
 I+++.+++ P.
 
-=== TO THE POOR AMONG FRIENDS, who are relieved by the charity and bowels of love, which God opens in other Friends towards them
+[.letter-heading]
```
Comment (maijapapaya) [inline unabridged-works-volume-4/original/17-letters.adoc:426]: I wasn't sure if this should be a letter heading or blurb!

File: unabridged-works-volume-4/original/17-letters.adoc [hunk 2]
```diff
@@ -651,11 +690,14 @@ but without the true, pure, fresh life.
 
 From a faithful friend and lover of souls.
```
Comment (maijapapaya) [inline unabridged-works-volume-4/original/17-letters.adoc:691]: SHould this be a [sscl]?

File: unabridged-works-volume-4/original/17-letters.adoc [hunk 3]
```diff
@@ -796,13 +842,16 @@ so gaping after the seat of government hath sunk the good thing in others also;
...
 that they cannot see the mighty breakings forth of the Spirit of Christ in his people.
 
+[.signed-section-signature]
 I+++.+++ P.
 
-=== TO RICHARD ROBERTS
+[.letter-heading]
+To Richrd Roberts
 
+[.salutation]
```
Comment (maijapapaya) [inline unabridged-works-volume-4/original/17-letters.adoc:851]: Is this a salutation?

File: unabridged-works-volume-4/original/17-letters.adoc (hunk 1 above)
Comment (jaredh159) [reply unabridged-works-volume-4/original/17-letters.adoc:426]: yeah, tough... but i think you did the right thing.

File: unabridged-works-volume-4/original/17-letters.adoc (hunk 2 above)
Comment (jaredh159) [reply unabridged-works-volume-4/original/17-letters.adoc:691]: I could go either way, but I think lean towards not making it a sscl, like you left it. 👍

**Review by jaredh159 — APPROVED** [review]
Looks really good. I spotted one typo, but I'm going to pull it down and experiment with those sort of unusual postscripts, so I’ll handle that.  Everything else looked good. Thanks!

**Pull request merged**